### PR TITLE
Feature/baer error control

### PR DESCRIPTION
**Pull request by fmirus**
this pull request includes:
- option to perform the controller step of cob_linear_nav frequently and update the goal with each topic callback instead of using actions
- some additional bugfixes in cob_linear_nav to avoid the "racing robot bug" when using linear nav with the ipa trajectory manager

### CODE REVIEW

**Comment by mgruhler** [comment]
visual review okay

Question: Should we bring this into ipa320?

To see the changes without the whitespace stuff, add ?w=1 at the end of the link in the diff window.

**Comment by fmirus** [comment]
@ipa-mig @abubeck if we don't want this in ipa320, is there a common branch (maybe abubeck/baer_prototype is a candidate) to develop in?

File: cob_linear_nav/src/cob_linear_nav.cpp [hunk 1]
```diff
@@ -78,319 +78,404 @@
...
+      move_base_msgs::MoveBaseGoal action_goal;
+
+      action_goal.target_pose = transformGoalToMap(*goal);
+      
+      action_client_->sendGoal(action_goal);
+      action_client_->stopTrackingGoal();
+    }
+    else
+    {
+      ROS_INFO("In ROS goal callback, using the PoseStamped as error and start control step.");
```
Comment (mgruhler) [inline cob_linear_nav/src/cob_linear_nav.cpp]: Better change the ROS_INFO to a ROS_DEBUG for not filling up the log directory

**Comment by abubeck** [comment]
@ipa-mig 's comment: +1 
Rest looks good.

**Comment by fmirus** [comment]
@ipa-mig changed ROS_INFO to ROS_DEBUG

**Pull request merged**